akka-docs/src/main/paradox/remoting.md

```diff
@@ -408,8 +410,8 @@ finished.
 
 @@@ note
 
-In order to switch off the logging, set
-`akka.remote.log-remote-lifecycle-events = off` in your
+In order to switch of the logging, set
```

Comment:
Isn't it to switch the logging `off`?

Reply:
Must have been tapping x during stand up :)